public/app/services/settingsService.js

```diff
@@ -50,4 +249,4 @@ angular.module('kibibitCodeEditor')
     return $(window).data('fullscreen-state');
   }
 
-}]);
+}]);
```

Comment:
@Thatkookooguy 
what does it mean? should I leave blank line at the end of the file?